### PR TITLE
Added check for trailing slash

### PR DESCRIPTION
**Pull request by markcurtis1970**
Removes the problems we can get with trailing slash requests to certain APIs.

### CODE REVIEW

File: cmd/local/conf/conf_test.go [hunk 1]
```diff
@@ -341,3 +341,12 @@ func TestConfReadingWhenLoggingParsingOfDdcYAML(t *testing.T) {
 	}
 	afterEachConfTest()
 }
+
+func TestURLsuffix(t *testing.T) {
```
Comment (rsvihladremio) [inline cmd/local/conf/conf_test.go:345]: add a test that has not trailing slash. need to verify we don't break anything (I know we don't but it's a good regression test)

Comment (markcurtis1970) [reply cmd/local/conf/conf_test.go:345]: Ok this is added into the same function - lmk if this looks good

Comment (rsvihladremio) [reply cmd/local/conf/conf_test.go:345]: that looks wrong ..expected shouldn't _add_ a / should it?

```
	testURL = "http://localhost:9047/some/path"
 	expected = "http://localhost:9047/some/path/"
 	actual = conf.SanitiseURL(testURL)
 	if expected != actual {
 		t.Errorf("\nexpected: %v\nactual: %v\n'", expected, actual)
```

Comment (markcurtis1970) [reply cmd/local/conf/conf_test.go:345]: sorry - typo 🤦

**Pull request merged**